source/ways-of-working/new-component/helm-chart.html.md.erb

```diff
@@ -109,6 +109,14 @@ redis:
         enabled: false
 ```
 
+## Advanced values for the preview environment
+
+We now have the ability to add additional value configurations for use within the preview environment which will be useful where teams need to test ccd/xui per PR. Historically this would have meant that there could be around 10-15 pods running per PR. To make use of this feature you need the following:
```

Comment:
1. don't use 'now' in documentation, it will be soon outdated.
2. use sentence per line, it makes it easier to review
3. avoid acronyms as much as possible, expand PR to pull request, if you add (PR) after the acronym you can use PR later.

Reply:
Hi Tim,

Should all now be resolved.

Reply:
screenshot please?

Reply:
Hi Tim,

Screenshot now attached:

![image](https://user-images.githubusercontent.com/33876448/127131541-9b83ed84-1d37-43c3-a947-445995fb30a2.png)